packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -122,7 +122,7 @@ const optionSets = {
 const filterModeDescription = `
 Controls the filtering behavior of the select. The default of \`none\` results in a dropdown with no input for filtering. A non-'none' setting results in a search input placed at the top or the bottom of the dropdown when opened (depending on where the dropdown is shown relative to the component). The \`standard\` setting will perform a case-insensitive and diacritic-insensitive filtering of the available options anywhere within the text of each option. The \`manual\` setting will provide the search input in the dropdown, but performs no filtering of the options. This allows for custom filtering to be implemented by the consuming application.
 
-It is recommended that if the select has 15 or fewer options that you use the \`none\` setting for the \`filter-mode\`.
+It is recommended that if the select has 15 or fewer options that you use the \`none\` setting for the \`filter-mode\`. It is recommended that each select should keep the same filter mode for its lifetime.
```

Comment:
I'll admit this guidance feels a little weird: there are other properties where the same guidance might apply but we don't bother to mention it. I considered other guidance like:
"Since filter mode affects behavior without any obvious indication to users, it is recommended not to change it after the select is first populated"

We could also just leave as-is and see if any other clients misinterpret the existing guidance. I'm open to reviewer feedback.